mqtt/mqtt-broker/src/session/mod.rs

```diff
@@ -74,6 +74,15 @@ impl Session {
         }
     }
 
+    pub fn client_info_omit_offline_sessions(&self) -> Option<&ClientInfo> {
```

Comment:
there is a method `Session::client_info` already. It didn't work for you in some cases?

Reply:
If it is Result and not Option, you can easily convert one to another: `session.client_info().ok()`

Reply:
I added this in order to explicitly filter out offline sessions until we deal with them correctly

Reply:
Actually it's unnecessary. I removed it